src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineView/bpp.jelly

```diff
@@ -62,13 +63,14 @@
         <j:if test="${hasBuildPermission}">
             <div class="icon-container">
             <j:if test="${from.isProjectParameterized()}">
-                <span class="pointer" onclick="buildPipeline.fillDialog('${app.rootUrl}${from.gridBuilder.firstJobLink}/build?delay=0sec', 'Starts the pipeline with parameters')">
+                <span class="bpp-trigger-pipeline-parameterized-params" data-fill-dialog-href="${app.rootUrl}${from.gridBuilder.firstJobLink}/build?delay=0sec" data-fill-dialog-title="Starts the pipeline with parameters" style="display: none" />
```

Comment:
Why the need for a different element to hold the data attributes? Is it not simpler to put them on the element on the next line?

Reply:
Got it - I made the changed and tested that it is working. I'll note all things things for future PR to avoid bloating the DOM.

Thank you!